CMakeLists.txt

```diff
@@ -1,16 +1,17 @@
-cmake_minimum_required(VERSION 3.10.2)
+cmake_minimum_required(VERSION 3.14)
```

Comment:
Please don't bump the version higher then the one available in the oldest supported Ubuntu release (currently 18.04). It makes packaging much more difficult.

https://packages.ubuntu.com/search?keywords=cmake

Reply:
I am aware of this, but Ubuntu 18.04 is about to turn EOL in April. It was necessary to update cmake so that libexpat can be located properly on Windows, which is a non-bundled dependency of APRutil now. Ubuntu 20.04 has 3.16 so that won't be an issue.